scanner/src/main/kotlin/Scanner.kt

```diff
@@ -141,15 +141,72 @@ class Scanner(
             }
         }
 
+        val filesByProvenance = (projectResults.files + packageResults.files)
```

Comment:
The name `filesByProvenance` is mis-leading me to think this is a map with `Provenance` as the key, which is not true. Should we simply name this `files`?